### ISSUE TITLE
aspnet core 2: Why is bower still in use when they clearly state we should move to yarn/webpack?

### ISSUE THREAD

**Issue by masterjs** [aspnet/Home#2086]
Hello,
I just took a look at the newest bits for aspnet core 2.0 preview and was surprised to see that Bower is still part of the default site template. It is working well, but the site clearly state: 

> .psst! While Bower is maintained, we recommend yarn and webpack for new front-end projects!

I was hoping the newest stuff would keep up with the current web world. Any information on that topic? (I did not found any information on that subject)

thank you,
JS

**Comment by davidfowl** [comment]
We aren't making any changes to this for 2.0. We'll likely revisit in a later release. You are free to use webpack or any other package manager today (we do it ourselves in other projects). As an example, the new SPA templates for react, and angular use webpack. 

/cc @SteveSandersonMS @madskristensen

**Comment by SteveSandersonMS** [comment]
I totally agree that Bower is no longer what you'd want in a new project, and that NPM/Webpack are established as the current standards for front-end package management and build. I'm sure our stock web projects will move in this direction as they already have for our SPA templates (Angular/React/React+Redux).

It's not yet clear that Yarn will displace NPM, and in fact NPM 5 brings most of the same benefits of Yarn (faster and fully deterministic restores with a lockfile) so it might well be that Yarn will move aside, having served its purpose of forcing the NPM client tool to modernise. This remains to be seen.

**Comment by masterjs** [comment]
Thank you for your answers. I just looked at the Angular template and I liked what I saw. Finally a web template in VS that also consider typescript and put the right build commands in the .csproj. I will surely use this stuff for my non-angular projects. I would recommend that you start putting recommendation in the MS documentation site that talks about bower that will will eventually be replaced.

thank you,
JS
